src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/pact/PactContractTest.kt

```diff
@@ -15,8 +15,6 @@ import org.springframework.context.annotation.Import
 import org.springframework.http.HttpHeaders
 import org.springframework.test.context.ActiveProfiles
 import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.common.config.JwtAuthHelper
-import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.domain.repository.CourseRepository
-
```

Comment:
For annoying things like empty lines, trailing commas, optimising imports, and so on, this repo also contains a helpful thing called `ktLintFormat`, which runs over the codebase and makes the relevant changes. You can also set it up as a pre-commit hook - instructions for it should be in the readme.

Reply:
Yeah, the build picked it up 👍 
Something for next time